.github/workflows/conventional-commits.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Check Conventional Commits'
+
+on:
+  pull_request_target:
```

Comment:
I think this is the best [trigger](https://github.com/amannn/action-semantic-pull-request#event-triggers) to use.

A quirk is that this check will only start being enforced on new PRs once this PR is merged into main branch